Foosball/Foosball/TournamentBracket.cs

```diff
@@ -14,24 +14,30 @@ namespace Foosball
     public partial class TournamentBracket : Form
     {
         private int _amount;
+        private List<string> _teamNames = new List<string>();
+        private int _round;
 
         public TournamentBracket(List<String> teamNames, int round = 1)
         {
             if(round == 1) {
                 var shuffledTeamNames = teamNames.OrderBy(a => Guid.NewGuid()).ToList();
                 teamNames = shuffledTeamNames;
             }
-            CheckIfEnded(teamNames);
+
+            _teamNames = teamNames;
+            _round = round;
+
+            CheckIfEnded(_teamNames);
             InitializeComponent();
             _amount = AmountFinder(teamNames.Count);
-            RemoveLabel(_amount);
-            AddTeamNames(teamNames, _amount, round);
+            RemoveLabel();
+            AddTeamNames(teamNames);
```

Comment:
If you create _teamNames, you can use it in other methods. Then there is no need to send it to method as parameter